### PR TITLE
Add Windows import workflows

### PR DESCRIPTION
**Pull request by adjackura**

### CODE REVIEW

File: daisy_workflows/image_import/README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ Imports a virtual disk file and converts it into a GCE image resource.
 Variables:
 * `source_disk_file`: A supported source virtual disk file, either local or in GCS.
 * `image_name`: The name of the imported image, will default to "imported-image-${ID}".
+* `image_name`: The size of the importer instance disk, additional disk space
```
Comment (zmarano) [inline daisy_workflows/image_import/README.md]: image_size?

Comment (zmarano) [inline daisy_workflows/image_import/README.md]: Or importer_instance_disk_size?

File: daisy_workflows/image_import/windows/metadata_scripts/run_startup_scripts.cmd [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@echo off
+REM Copyright 2015 Google Inc. All Rights Reserved.
```
Comment (zmarano) [inline daisy_workflows/image_import/windows/metadata_scripts/run_startup_scripts.cmd]: 2017

**Comment by adjackura** [comment]
The binary is our metadata scripts runner, I preferred to commit it instead of having the migration script pull it out of the bootstrap instance in order to keep the translation somewhat hermetic.

**Pull request merged**